habitat_baselines/utils/gym_definitions.py

```diff
@@ -22,9 +22,11 @@
 HABLAB_INSTALL_PATH = "HABLAB_BASE_CFG_PATH"
 
 if HABLAB_INSTALL_PATH in os.environ:
-    base_dir = os.environ[HABLAB_INSTALL_PATH]
+    config_base_dir = os.environ[HABLAB_INSTALL_PATH]
```

Comment:
I think this `config_base_dir` should live in core habitat and not in gym since it is the HABLAB_INSTALL_PATH.
Leaving is in gym could lead to strange imports (like someone needing to import gym_definitions just to get the path to hab-lab (I believe there is an instance of this in the colab notebook)